### PR TITLE
Feature/22 prepare demo page

### LINKED ISSUE #22: Generate and publish demo
For publishing we might need permission to do it to zalando-incubator

### PR DESCRIPTION
**Pull request by dami-gg**
Closes #22

### CODE REVIEW

File: src/app/navigation.service.js [hunk 1]
```diff
@@ -30,50 +32,106 @@ class NavigationService {
...
    */
   handleBusinessPartner() {
-    const accessibleBusinessPartners = this.stateHandlerService.getProperty('businessPartners', 'accessibleBusinessPartners');
+    const accessibleBusinessPartners =
+            this.stateHandlerService.getProperty('businessPartners', 'accessibleBusinessPartners');
 
     if (accessibleBusinessPartners.length === 0) {
-      // TODO Redirect to User Management? and log error
+      // TODO Log error
+      // TODO Redirect to user management page?
```
Comment (frontendherodk) [inline src/app/navigation.service.js]: Would be nice if we could redirect to an error page with a custom message eg. "It looks like you don't have any accessible business partners"

Comment (dami-gg) [inline src/app/navigation.service.js]: yes... too nice for the current possibilities we have :D I'd say we address all those TODOs together as final task

File: src/app/navigation.service.js [hunk 2]
```diff
@@ -30,50 +32,106 @@ class NavigationService {
...
    * @param businessPartnerId
-   * @returns {boolean}
+   * @returns {Object}
    */
-  isValidBusinessPartnerForApplication(accessibleBusinessPartners, businessPartnerId) {
-    return accessibleBusinessPartners.filter((businessPartner) => {
-        return businessPartner.id === businessPartnerId;
-      }).length > 0;
+  findBusinessPartnerAmongAccessible(accessibleBusinessPartners, businessPartnerId) {
+    const found = accessibleBusinessPartners.filter((businessPartner) => {
```
Comment (dami-gg) [inline src/app/navigation.service.js]: I'd love to do that as well but that's an experimental feature of ES6 which is not supported by Karma and will make the tests fail

**Comment by dami-gg** [comment]
:+1:

File: src/app/navigation.service.js (hunk 2 above)
Comment (frontendherodk) [inline src/app/navigation.service.js]: As far as I can see it's actually a part of the specs for ES2015, but if it doesn't work with Karma then I'll approve it for now. Maybe we can look into a solution for the future. I would be sad that we're limiting ourselves

**Comment by frontendherodk** [comment]
👍

**Pull request merged**